CONTRIBUTING.md

```diff
@@ -16,56 +16,58 @@ reading through the demo code repositories in [dwave-examples](https://github.co
 ## Steps for Contributing
 
 1. Follow GitHub's instructions for [creating a repository from a template](https://docs.github.com/en/repositories/creating-and-managing-repositories/creating-a-repository-from-a-template).
-   Alternatively, clone this repository (or one of our [existing examples](https://github.com/dwave-examples)) as a
+   Alternatively, clone this repository as a
    starting point from which to implement your example.
-2. Implement your example by following the instructions in the [README](README.md).
-3. [Open an issue](https://github.com/dwave-examples/template-dash/issues/new/choose) in the
-   [dwave-examples/template-dash](https://github.com/dwave-examples/template-dash) repository with a link to your example.
 
-4. To add your example, D-Wave will fork your repository. We may make a pull
-   request if we would like your approval for certain code changes.
+2. Implement your example by following these instructions:
```

Comment:
Rather than have two levels of instructions, which gives an impression of complexity, consider using separate points for these: implementation guidelines

2. statement on files
3. statement on directories 
4. readme

Reply:
Updated: https://github.com/dwave-examples/template-dash/pull/3/commits/a8faa217776de94309780459ad81de2591cf0f58

Reply:
I personally find it more confusing to have it all in one line, and this goes for any documentation I read, so my preference would be to keep it as is but let me know how strongly you feel about this.